src/observables/FromEventPatternObservable.ts

```diff
@@ -0,0 +1,72 @@
+import Scheduler from '../Scheduler';
+import Observable from '../Observable';
+import Subscription from '../Subscription';
+import tryCatch from '../util/tryCatch';
+import {errorObject} from '../util/errorObject';
+
+export default class FromEventPatternObservable<T, R> extends Observable<T> {
+
+  static create<T, R>(addHandler: (handler: Function) => void, removeHandler: (handler: Function) => void, selector?: (x: R) => T, scheduler?: Scheduler) {
+    return new FromEventPatternObservable(addHandler, removeHandler, selector, scheduler);;
```

Comment:
why is a scheduler required here unless to allow someone to change where the unsubscribe happens (which is where subscribeOn) was meant to address